### PR TITLE
Implementation of web tokens (based on rA)

### PR DESCRIPTION
**Pull request by csnv**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Implementation of webtokens required for webservices in newer clients, based on rAthena implementation.

This change is proposed to allow third party applications to bring the required functionality for newer clients, as a temporal measure.

This is **not** a replacement of the ongoing work of 4144 as published here https://board.herc.ws/topic/20151-http-support-in-hercules-funding/

**Issues addressed:** 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by csnv** [comment]
Now it should be good enough for reviewing. Kindly re-run those failed checks, please!

**Comment by 4144** [comment]
this pr do minimal code base changes, may be better create it as plugin?

File: src/login/packets_ac_struct.h [hunk 1]
```diff
@@ -83,8 +83,7 @@ struct PACKET_AC_ACCEPT_LOGIN {
 	char last_login_time[26]; ///< Last login timestamp
 	uint8 sex;                ///< Account sex
 #if PACKETVER >= 20170315
-	char twitter_auth_token[16];
-	uint8 twitter_flag;
+	char web_auth_token[WEB_AUTH_TOKEN_LENGTH];
```
Comment (4144) [inline src/login/packets_ac_struct.h:86]: here error. token size is only 16. after should be one byte flag

**Review by 4144 — CHANGES_REQUESTED** [review]

**Comment by 4144** [comment]
also better sql files changes in separate commit

**Comment by csnv** [comment]
> this pr do minimal code base changes, may be better create it as plugin?

Indeed, it makes more sense to have this as a plugin. I'll look into it and take your other comments into consideration.

Thanks a lot!

**Pull request closed without merging**